pages/test.tsx

```diff
@@ -0,0 +1,29 @@
+import { useGetContributionsQuery, useGetMembersQuery, useGetUserQuery, useGetUserActiveTaskQuery } from '@/src/services/serverApi'
+import React from 'react'
+import { wrapper } from '@/src/store';
+
+interface Member {
+  [key: string]: string;
+}
+
+interface MyProps {
+  members: Member;
+}
+
+function Test(props:MyProps) {
+  // const { data : members } = useGetMembersQuery(1);
+  const { members } = props;
+  const { data : user } = useGetUserQuery("satyam-bajpai");
```

Comment:
why are we sending your name here ?

Reply:
Used it for testing purposes😅.
Removed now